stan/math/prim/eigen_plugins.h

```diff
@@ -214,5 +214,17 @@ inline CwiseUnaryView<vi_Op, Derived>
 vi() { return CwiseUnaryView<vi_Op, Derived>(derived());
 }
 
+template <typename CustomTernaryOp, typename OtherDerived1,
+          typename OtherDerived2>
+EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE
+const CwiseTernaryOp<CustomTernaryOp, const Derived,
+                     const OtherDerived1, const OtherDerived2>
+  ternaryExpr(const OtherDerived1& other1, const OtherDerived2& other2,
+              const CustomTernaryOp& func = CustomTernaryOp()) const {
```

Comment:
Is there a reason this has to have a default value?

Reply:
No reason, was just following the Eigen code: https://gitlab.com/libeigen/eigen/-/blob/master/Eigen/src/Core/CwiseTernaryOp.h#L123

Reply:
Seems pretty unnecessary, so reverted!